torchvision/models/segmentation/deeplabv3.py

```diff
@@ -46,9 +46,9 @@ class DeepLabV3(_SimpleSegmentationModel):
 
 
 class DeepLabHead(nn.Sequential):
-    def __init__(self, in_channels: int, num_classes: int) -> None:
+    def __init__(self, in_channels: int, num_classes: int, atrous_rates: List[int] = [12, 24, 36]) -> None:
```

Comment:
We should avoid setting mutables as default values in general as there's a risk that we will change that value inplace, and subsequent calls would be using the wrong default.

```suggestion
    def __init__(self, in_channels: int, num_classes: int, atrous_rates: Sequence[int] = (12, 24, 36)) -> None:
```

Reply:
Yeah, makes sense. I thought the same, therefore had made it tuple before. 😅

Reply:
Updated the PR.